modules/images/fetchpriority/load.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Module Name: Fetchpriority
+ * Description: Adds `fetchpriority=high` parameter to the non lazy loaded images.
+ * Experimental: Yes
+ *
+ * @since n.e.x.t
+ * @package performance-lab
+ */
+
+/**
+ * Filter image tags in content to add fetchpriority to the image tag if lazy loaded is not set.
+ *
+ * @since n.e.x.t
+ *
+ * @param string $filtered_image The filtered image.
+ * @param string $context        The context of the image.
+ *
+ * @return string image tag
+ */
+function fetchpriority_img_tag_add( $filtered_image, $context ) {
+
+	if ( 'the_content' !== $context ) {
+		return $filtered_image;
+	}
+
+	if ( strpos( $filtered_image, 'loading="lazy"' ) === false ) {
+		$filtered_image = str_replace( '<img ', '<img fetchpriority="high" ', $filtered_image );
+		remove_filter( 'wp_content_img_tag', 'fetchpriority_img_tag_add', 10, 2 );
```

Comment:
See above. Also:
* `remove_filter()` only supports three attributes.
* We need to remove the other filter for featured image which I suggested here too so that it's really always only on the first image it encounters.

```suggestion
		remove_filter( 'wp_content_img_tag', 'fetchpriority_img_tag_add_attr', 10 );
		remove_filter( 'post_thumbnail_html', 'fetchpriority_filter_post_thumbnail_html' );
```